api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
```

Comment:
Could it be because you didn't have catch blocks on your parent promise chain in the controller?